server.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express();
+const port = 8080;
+var fs = require('fs');
+//bootup time load data
+var jsonContent = fs.readFileSync('./data.json', 'utf-8');
+var jsonDataPut = null;
+app.get('/', (req, res) => {
+    // json file data fatching
+    var jsonDataGet = jsonContent;
+    console.log(jsonDataGet);
+    res.send(jsonDataGet);
+})
+app.put('/', (req, res) => {
+    jsonDataPut = fs.readFileSync('./data.json','utf-8');
```

Comment:
jsonDataPut

Do you need this variable ?

Reply:
Sir, we can use directly jsonContent is put method also. I modified it in code.